newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js

```diff
@@ -258,29 +261,25 @@ export default class RenderedCustomObjectInstance extends Rendered3DInstance
     }
   }
 
-  getRendererOfInstance = (instance: gdInitialInstance) => {
+  getRendererOfInstance = (
+    instance: gdInitialInstance,
+    customObjectConfiguration: gdCustomObjectConfiguration
```

Comment:
What's the reason for this? More performant? Was this useless to fetch the configuration again from the events based object?

Reply:
The event-based object children configurations are like default properties values where the custom object configuration is the actual property values.
In an example with a EBO Player with blue placeholders, I added a 2nd custom object with red resources. Without the fix, the instances where still blue.